include/frozen/bits/pmh.h

```diff
@@ -55,7 +55,7 @@ struct pmh_buckets {
 
   using bucket_t = cvector<std::size_t, bucket_max>;
   carray<bucket_t, M> buckets;
-  uint64_t seed;
+  size_t seed;
```

Comment:
In the default hash algorithm, the seed is casted to size_t, so that's not an issue. Still, I agree with you that the PRNG still needs to use a stable type where we know all the properties. i'll update that!